core/object/worker_thread_pool.cpp

```diff
@@ -780,10 +780,20 @@ void WorkerThreadPool::init(int p_thread_count, float p_low_priority_task_ratio)
 
 	threads.resize(p_thread_count);
 
+	Thread::Settings settings;
+#ifdef __APPLE__
```

Comment:
If Apple should use a large stack size for SPIRV-Cross, I think it would be good to add a short comment here explaining the deviation from other platforms.

Reply:
Done – good call.